UsingInPipelines.md

```diff
@@ -0,0 +1,64 @@
+# Examples of Pipelines
```

Comment:
Do we have a link to this MD file somewhere to make it more easily discoverable?

Reply:
this file is added to this branch by mistake, removing.